### DISCUSSION TITLE
Language

### DISCUSSION THREAD

**Discussion by securitymonster** [chris2511/xca#541] [General]
Had to reinstall XCA and suddenly it is in Dutch. My whole computer is in English and I prefer English. Is there a way to change the language because I can't find it.

**Comment by securitymonster** [comment]
It seems really tied to the region settings in Mac OS system preferences. If I have:

1. English (primary)
2. Dutch

XCA is in Dutch

If I have:
1. English (primary)

XCA is in a language I don't recognise

If I have:
1. English (primary)
2. English US

XCA is in a language I don't recognise.

**Comment by chris2511** [comment]
Please see #532
You can set english in the File Menu. See: https://github.com/chris2511/xca/issues/532#issuecomment-1967466586

**Reply by securitymonster** [reply]
Thanks, didn't find that thread when I searched for a solution, my bad.
